lib/samples/show_device_location_history/show_device_location_history_sample.dart

```diff
@@ -160,19 +180,44 @@ class _ShowDeviceLocationHistorySampleState
     });
```

Comment:
I don't think we need `_statusSubscription` at all. There's nothing in the UI that gets updated when the status changes.